app/admin_statistics.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python2.7
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import sys
+
+import model
+from utils import *
+
+class Handler(BaseHandler):
+    repo_required = False
+    admin_required = True
+
+    def get(self):
+        """
```

Comment:
Again, you don't need to document the implementation (unless it's too complicated or tricky). You won't need a comment for this method because it's rather obvious what get() method is for a Handler.

Instead, it may be good to have a comment for the class Handler, to explain what this page is.